tools/cli/internal/openapi/openapi.go

```diff
@@ -1,13 +1,71 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package openapi
 
+//go:generate mockgen -destination=../mocks/mock_openapi.go -package=mocks github.com/mongodb/openapi/tools/cli/internal/openapi Parser,Merger
```

Comment:
let's avoid some of the CLI mistakes by creating "common" folders and follow a better collocation pattern, let's put the mock next to the file it mocks